src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(readData(fromFileName));
+        writeToFile(report, toFileName);
+    }
+
+    private int[] readData(String fromFileName) {
```

Comment:
this method not only reads the file but also performs calculations for the report. It's a violation of SPR. this method should only read a file and return a list of strings

Reply:
What is SPR? Strategic Petroleum Reserve? Society for Psychotherapy Research? Surface Plasmon Resonance? Were we learned this yet? If yes, please send me a link to a topic, I`ll check, but if we weren't... Ok, send it anyways

Reply:
But also read file and calc data in different places is a good idea anyways.